### DISCUSSION TITLE
How to change the pinout

### DISCUSSION THREAD

**Discussion by omgitsdango** [manna-harbour/miryoku#338] [Q&A] [answered]
I dont find the file where the pinout of my nicenanov2 is defined. I want to build miryoku zmk for my dactyl manuform 4x5 but the keys are not working or just plain wrong. Am i missing something?

**Comment by omgitsdango** [comment]
Okay, found the reference to the shield and board in the .github/workflows/outboards folder.
Is it possible to add my own shield and board somehow? My dactyl is wired by myself.

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku_zmk#supported-keyboards.
